tests/runtime/watchpoint

```diff
@@ -1,7 +1,8 @@
 NAME watchpoint - absolute address
-RUN {{BPFTRACE}} -e 'watchpoint:0x10000000:8:w { printf("hit!\n"); exit() }' -c ./testprogs/watchpoint
+RUN install -m 644 /dev/null /tmp/watchpoint_mem; ./testprogs/watchpoint & > /dev/null; {{BPFTRACE}} -e "watchpoint:$(awk '{print $1}' /tmp/watchpoint_mem):8:w { printf(\"hit!\n\"); exit() }"
```

Comment:
Hmm, by removing the `-c` param, this turns it into a kernel watchpoint, no?

Docs say:
> In the first form, an absolute address is monitored. If a pid (-p) or a command (-c) is provided, bpftrace takes the address as a userspace address and monitors the appropriate process. If not, bpftrace takes the address as a kernel space address.

Reply:
This still applies, right? It seems to me that it would be better to run `testprogs/watchpoint` from the `BEFORE` clause and then use `-p {{BEFORE_PID}}` not to transform this into a kernel watchpoint.

Reply:
My bad, I missed that. Now implemented.